src/App.tsx

```diff
@@ -1,73 +1,30 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
+// import { peopleFromServer } from './data/people';
+import Dropdown from './Dropdown';
+import { Person } from './types/Person';
 import { peopleFromServer } from './data/people';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [person, setPerson] = useState<Person | null>(null);
+  const { name, born, died } = person || {};
+
+  const onSelected = (personArg: Person | null) => {
```

Comment:
```suggestion
  const handlePersonSelected = (personArg: Person | null) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364